drivers/sub8_driver_utils/camera_utils/src/dc1394_reset_bus.c

```diff
@@ -0,0 +1,67 @@
+//
```

Comment:
I think it's because he pushed some changes from his computer directly from the sub account, in between when I first opened this PR and now.